tests/unit/unit1607.c

```diff
@@ -67,34 +70,37 @@ static const char skip = 0;
 static const struct testcase tests[] = {
   /* spaces aren't allowed, for now */
   { "test.com:80:127.0.0.1, 127.0.0.2",
-    "test.com", 80, { NULL, }
+    "test.com", 80, true, { NULL, }
```

Comment:
We always use `TRUE` in uppercase in the code.

Reply:
I can change the `true` and `false` to upper case.
I used lower case because the code I changed (`Curl_loadhostpairs`) already used lower case `true` and `false` so I thought that was the standard or at least OK. Should I also change the existing lower case `true`/`false` in that function?

Re. changing the docs: they should state the min. version that supports the new syntax. Should I just put 7.75 there?

And last question: do you prefer fixup commits or is it OK if I simply amend the commit?

Reply:
On `true` vs `TRUE`: if you want to, you can update the other code too but that would be a separate change and would need to be a separate PR so maybe save that? (I'll possibly also check if I can expand `checksrc` to detect and warn for this)

I personally like amended commits and force-pushes (as long as you're the single developer in the branch) so that we can review the final result easier.